.circleci/config.yml

```diff
@@ -1,8 +1,18 @@
 version: 2.1
 
 orbs:
-  codacy: codacy/base@2.0.1
-  codacy_plugins_test: codacy/plugins-test@0.7.1
+  codacy: codacy/base@2.7.0
+  codacy_plugins_test: codacy/plugins-test@0.10.3
+  coverage-reporter: codacy/coverage-reporter@7.5.0
+
+jobs:
+  send-coverage-report:
+    machine: true
+    working_directory: ~/workdir
+    steps:
+      - attach_workspace:
+          at: ~/workdir
+      - coverage-reporter/send_report
 
```

Comment:
I think we can expose bot a job and an executor in the coverage reporter orb

Reply:
In this case I prefer much more using the script directly instead of having to create an entire job

Reply:
When doing the orb, we decided to create a command. But creating a job just to send the coverage is indeed a not very nice solution.
I think we should indeed expose a job and an executor in the coverage reporter orb so we don't need this.